eng/Signing.props

```diff
@@ -13,11 +13,14 @@
     <FileSignInfo Include="System.dll" CertificateName="3PartySHA2" />
   </ItemGroup>
 
-  <ItemGroup>
-    <!-- For Selenium.WebDriver -->
-    <ItemsToSign Remove="@(ItemsToSign->WithMetadataValue('Filename', 'WebDriver'))" />
-    <ItemsToSign Remove="@(ItemsToSign->WithMetadataValue('Filename', 'Newtonsoft.Json'))" />
-  </ItemGroup>
+  <!-- We need  this to be inside a target to workaround: https://github.com/microsoft/msbuild/issues/5445 -->
+  <Target Name="PrepareItemsToSign" BeforeTargets="Sign">
+    <ItemGroup>
+      <!-- For Selenium.WebDriver -->
+      <ItemsToSign Remove="@(ItemsToSign->WithMetadataValue('Filename', 'WebDriver'))" />
+      <ItemsToSign Remove="@(ItemsToSign->WithMetadataValue('Filename', 'Newtonsoft.Json'))" />
+    </ItemGroup>
+  </Target>
 
   <PropertyGroup>
```

Comment:
@radical can you please remove the property group around `AllowEmptySignList` below?

I think we don't want that behavior and it would have warned us before that we are removing everything.

@MattGal or you remember why we have it?

Reply:
@premun sorry I don't ; other once something works whether it's right or not it tends to be left alone.  Are you saying signing never worked?

Reply:
@MattGal not that - something else happened - because of a bug in MSBuild, @radical removed all `ItemsToSign` instead of just the ones he didn't want to get signed and our PR build then went green and we didn't notice nothing got signed actually. And then after merging, our internal CI failed on signing validation. So I was thinking we get rid of this (I checked and for example arcade doesn't have this set) and catch these type of mistakes earlier